{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "52d999bf_24c1c85a",
        "filename": "src/chromiumos/tast/remote/bundles/cros/syzcorpus/syzutils/utils.go",
        "patchSetId": 1
      },
      "lineNbr": 103,
      "author": {
        "id": 1118346
      },
      "writtenOn": "2021-06-24T03:30:39Z",
      "side": 1,
      "message": "Do we have a use case where we want to keep the process running after timeout? If not, I think it\u0027s better to make RunRepro responsible for killing the process at the end, rather than introducing another method to kill the process.",
      "range": {
        "startLine": 103,
        "startChar": 5,
        "endLine": 103,
        "endChar": 13
      },
      "revId": "31b676bce3f4e7cc68a03389be4033822514d644",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "2f8ddb4a_92a419c3",
        "filename": "src/chromiumos/tast/remote/bundles/cros/syzcorpus/syzutils/utils.go",
        "patchSetId": 1
      },
      "lineNbr": 108,
      "author": {
        "id": 1118346
      },
      "writtenOn": "2021-06-24T03:30:39Z",
      "side": 1,
      "message": "Hmm, yes, I remembered that the remote process may not be terminated even if it hits timeout due to b/187790213.\n\nYou might want to try timeout(1) instead. It\u0027s easier and safer than trying to find the process by pkill and ps later.",
      "range": {
        "startLine": 108,
        "startChar": 1,
        "endLine": 108,
        "endChar": 51
      },
      "revId": "31b676bce3f4e7cc68a03389be4033822514d644",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}